### PR TITLE
Add personal_note actions + tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/personal_note/create.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        * set meeting_id from content_object_id
+        """
+        if self.auth.is_anonymous(self.user_id):
+            raise ActionException("Can't create personal note for anonymous")
+        instance["user_id"] = self.user_id
+        if not (instance.get("star") or instance.get("note")):
+            raise ActionException("Can't create personal note without star or note.")
+        fqid = instance["content_object_id"]
+        content_object = self.datastore.get(fqid, ["meeting_id"])
+        instance["meeting_id"] = content_object.get("meeting_id")
```
Comment (jsangmeister) [inline openslides_backend/action/personal_note/create.py:36]: Please use `CreateActionWithInferredMeetingMixin` for this

File: openslides_backend/action/personal_note/create.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        return instance
```
Comment (jsangmeister) [inline openslides_backend/action/personal_note/create.py:37]: Check missing that `user_id` and `content_object_id` are unique together -> please add it with test

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/personal_note/create.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/personal_note/create.py]: Done.

File: openslides_backend/action/personal_note/create.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/personal_note/create.py:51]: Add a check for (user_id, content_object_id) unique. Add a test.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**